site2/docs/cookbooks-deduplication.md

```diff
@@ -70,7 +74,7 @@ The instructions for Java, Python, and C++ clients are different.
 <!--DOCUSAURUS_CODE_TABS-->
 <!--Java clients-->
 
-To enable message deduplication on a [Java producer](client-libraries-java.md#producers), set the producer name using the `producerName` setter, and set the timeout to `0` using the `sendTimeout` setter. 
+Not to break the guarantee order on a [Java producer](client-libraries-java.md#producers) sending to a topic with message deduplication active, set the producer name using the `producerName` setter, and set the timeout to `0` using the `sendTimeout` setter. 
```

Comment:
what is the meaning of "Not to break the guarantee order on a Java producer sending to a topic with message deduplication active"? 

what are the logical relationships between the 3 parts of this whole sentence?

would u mind re-writing this sentence? Thanks

Reply:
```suggestion
To ensure the guarantee order on a [Java producer](client-libraries-java.md#producers) sending to a topic with message deduplication enabled, set the producer name using the `producerName` setter, and set the timeout to `0` using the `sendTimeout` setter. 
```

@AlvaroStream did you want to say something like this?